embassy-rp/src/lib.rs

```diff
@@ -402,6 +404,8 @@ embassy_hal_internal::peripherals! {
 
     WATCHDOG,
     BOOTSEL,
+
+    TRNG
```

Comment:
you should cfg this for rp235x only

Reply:
I think this whole peripherals! macro call is under #[cfg(feature = "_rp235x")]
Is that the right cfg?

Reply:
ah my bad, didn't notice. sorry!